docs/overview/script-storage.md

```diff
@@ -1,28 +1,27 @@
 ---
 title: Office Scripts file storage and ownership
 description: Information about how Office Scripts are stored in Microsoft OneDrive and transferred between owners.
-ms.date: 09/08/2023
+ms.date: 10/20/2023
 ms.localizationpriority: medium
 ---
 
 # Office Scripts file storage and ownership
 
-Office Scripts are stored as **.osts** files in your Microsoft OneDrive or a SharePoint folder. They are stored separately from a workbook. To give users who are outside of the SharePoint site access to the script, [share the script with an Excel workbook](excel.md#share-office-scripts). This means you're linking the script with the file, not attaching it. Whoever has access to the Excel file will also be able to view, run, or make a copy of the script.
+The details of how scripts are stored and shared depend on your Microsoft 365 subscription. Select the relevant tab to learn more.
 
-Excel only recognizes and runs a script if it's in your OneDrive folder, a Sharepoint folder, or shared with the workbook. This means Excel needs internet connectivity to access Office Scripts.
+## File storage
 
-> [!NOTE]
-> Admin settings for Conditional Access in OneDrive and SharePoint affect Office Scripts. For more information, see the [Conditional Access section of Platform limits and requirements with Office Scripts](../testing/platform-limits.md#conditional-access).
+# [For business/education](#tab/business)
 
-## OneDrive
+Office Scripts are stored in your OneDrive by default. The **.osts** files are found in the **/Documents/Office Scripts/** folder. Any edits made to these **.osts** files, such as renaming or deleting files, will be reflected in the Code Editor and Script Gallery. Excel only recognizes and runs a script if it's in your OneDrive folder, a Sharepoint folder, or shared with the workbook. This means Excel needs internet connectivity to access Office Scripts.
 
-The default behavior is that Office Scripts are stored in your OneDrive. The **.osts** files are found in the **/Documents/Office Scripts/** folder. Any edits made to these **.osts** files, such as renaming or deleting files, will be reflected in the Code Editor and Script Gallery.
+### OneDrive
```

Comment:
OneDrive for Business (to help disambiguate the enterprise/consumer cases?)

Reply:
This header is only in the "business" tab. Did you view this on the Learn staging site?

Reply:
Ah, gotcha.  No, I'm looking at the raw markdown.